raiden/tests/scenarios/bf1_basic_functionality.yaml

```diff
@@ -205,6 +205,17 @@ scenario:
             - wait: 100
             - assert_sum: {from: 0, balance_sum: 1_895_000_000_000_000_000}
             - assert_sum: {from: 3, balance_sum: 2_100_000_000_000_000_000}
+      - serial:
+          name: "Make 100 payments from 3 to 0"
+          repeat: 100
+          tasks:
+            - transfer: {from: 3, to: 0, amount: 1_000_000_000_000_000, lock_timeout: 30}
+      - serial:
+          name: "Assert after 100 payments from 3 to 0"
+          tasks:
+            - wait: 100
+            - assert_sum: {from: 0, balance_sum: 1_105_000_000_000_000_000}
```

Comment:
If the assert moves here then it needs to be adjusted to also take into account the balance of the channel `0-4` again.

The previous balance was `1_995_000_000_000_000_000` I believe

if I am to judge by this commit 265ff6eb4208e7096db1ca64c50db7b6c0654d18

You can also remove the 40 blocks waiting that I added to make sure the channel is settled at the assertion time.